discord/user.py

```diff
@@ -151,6 +154,11 @@ def _update(self, data: UserPayload) -> None:
             self.nameplate = Nameplate(data=nameplate, state=self._state)
         else:
             self.nameplate = None
+        primary_guild = data.get("primary_guild", None)
+        if primary_guild and primary_guild.get("identity_enabled"):
```

Comment:
What's the point of having identity_enabled on the primary guild object if this is here

Reply:
edge case https://canary.discord.com/channels/881207955029110855/903819190274764800/1409029789292957706